### DISCUSSION TITLE
Error handling / logging

### DISCUSSION THREAD

**Discussion by ajlardner** [SteveG/underthebar#9] [Ideas]
Trying to use the app mostly to create new routines from JSON as sort of a manual import from other apps and spreadsheets I've used. My routine is not saving but I can't see why as I can't find any error handling or logging. Suggesting that any errors are somehow displayed to the user as they happen, as well as being logged to some kind of log file (if that does not already happen). 

I would be happy to take a stab at this and open a PR if you think it is valuable work.

**Comment by Damien-Ergun** [comment]
Hi, great app but I am not able to sign in to hevy on the pop up with my google account. Am I the only one it happens to or is it something that should be corrected ?

**Reply by SteveG** [reply]
I've always used a hevy username/password so I can't test that at all sorry. Other people that had used google were able to use the Hevy password change function to generate a username/password. But if you just close the first login dialog you should get a second dialog that would give the option to take details from a firefox cookie automatically, or you can enter the details from a cookie yourself.  So that is another option - log in to the Hevy web app in firefox and then use that function.
